### ISSUE TITLE
badge buttons have been reverted to default color (react-bootstrap breaking changes from v1.x)

### ISSUE THREAD

**Issue by tomastuzikas** [fwouts/prmonitor#1050]
Hello,

react-bootstrap api has been updated:
- https://react-bootstrap.netlify.app/migrating/#summary-of-breaking-changes-from-v1x

## main issues:

- badge buttons have lost their color (default blue is shown)
https://react-bootstrap.netlify.app/migrating/#badge

changes needed:
```html
  <SpacedBadge pill variant="danger" key="unreviewed">
to
  <SpacedBadge pill bg="danger" key="unreviewed">
etc.
```

- Dropdown component prop error in console
https://react-bootstrap.netlify.app/migrating/#dropdown

changes needed:
```js
  <InlineDropdown onClick={preventDefault} alignRight>
to
  <InlineDropdown onClick={preventDefault} align="end">
```

screenshot:
![image](https://user-images.githubusercontent.com/25122240/195940283-7e5df73a-cb1a-4729-8ee5-aebd76af06b2.png)

----

Also, current webpack-cli version throws incompatibility errors:
```
"webpack": "^5.74.0",
"webpack-bundle-analyzer": "^4.6.1",
"webpack-cli": "^4.10.0",
"webpack-dev-server": "^4.10.0"
```

i.e.:
```
[webpack-cli] Invalid options object. Dev Server has been initialized using an options object that does not match the API schema.
 - options has an unknown property 'watchContentBase'. These properties are valid:
   object { allowedHosts?, bonjour?, client?, compress?, devMiddleware?, headers?, historyApiFallback?, host?, hot?, http2?, https?, ipc?, liveReload?, magicHtml?, onAfterSetupMiddleware?, onBeforeSetupMiddleware?, onListening?, open?, port?, proxy?, server?, setupExitSignals?, setupMiddlewares?, static?, watchFiles?, webSocketServer? }
```

I had to update devServer to webpack 5 properties and it worked (just interested if it works fine for others):
```js
  devServer: {
    static: {
        directory: path.join(__dirname, 'public'),
        watch: true,
    },
    compress: true,
    watchContentBase: true,
    hot: true,
    port: 9000,
  },
```

**Comment by fwouts** [comment]
Thanks for the report! Feel free to make a PR and I'll release it.

**Comment by fwouts** [comment]
Fixed by #1051 in v0.10.6.
